src/components/users/LoginForm/login-form.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../style/common/constants";
+@import "../../../style/common/typography";
+
+.login-form {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  align-items: center;
+  align-content: center;
+}
+
+.login-form-input-container {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 3vh;
+}
+
+.login-form-input {
+  border-radius: 8px;
+  box-sizing: border-box;
+  height: 4vh;
+  margin-top: 1vh;
```

Comment:
Do you really need to use `vh` unit for margins here? Take in count that `vh` is really volatil since it depends on the height of the user's viewport, it's not an absolute unit like `px`.

Reply:
If I want to make it dependant, it's ok?
Anyway I'll change this after I add all the elements in the div